### ISSUE TITLE
Carthage Support

### ISSUE THREAD

**Issue by mingsai** [hackiftekhar/IQMediaPickerController#14]
Any chance you could add support for Carthage? Currently when trying to pull, one is confronted with the following message:

No tagged versions found for github "hackiftekhar/IQMediaPickerController"

Which implies it might be as easy as adding some sort of tag. I am not certain of the requirement to support Carthage but that initial message seems as though adding support might be trivial. I'd prefer Carthage because the overhead with cocoa pods  is something I'd like to avoid and I am used to adding additional frameworks to the IDE (as others may be as well).

Found on SO that one can simply add the branch to the Cartfile. (http://stackoverflow.com/questions/31069315/carthage-errors-no-tagged-versions-found-for-github)
# Cartfile

github "hackiftekhar/IQMediaPickerController" "master"

However this led to an error:
Dependency "IQMediaPickerController" has no shared framework schemes
# Turns out that SO had this to say

Basically the shared scheme is not being added to the git repo.
(http://stackoverflow.com/questions/35054788/carthage-no-shared-framework-schemes-for-ios-platform-for-my-own-framework)
# Framework Support

Medium actually has a nice article on making a Carthage framework (https://medium.com/@filippotosetto/how-to-create-a-carthage-framework-8d9d65f98ac2#.9ger2pyn9)
# Shared Framework Scheme

Found an option to share Scheme on the Edit Scheme dialog box (Product > Edit Scheme)

**Comment by hackiftekhar** [comment]
I'll try to work on this and will create a framework target for Carthage support.
